x-pack/plugins/security_solution/server/endpoint/services/artifacts/manifest_manager/manifest_manager.mock.ts

```diff
@@ -23,22 +24,29 @@ import {
 
 export enum ManifestManagerMockType {
   InitialSystemState,
+  ListClientPromiseRejection,
   NormalFlow,
 }
 
 export const getManifestManagerMock = (opts?: {
   mockType?: ManifestManagerMockType;
   cache?: LRU<string, Buffer>;
+  exceptionListClient?: ExceptionListClient;
   packageConfigService?: jest.Mocked<PackageConfigServiceInterface>;
   savedObjectsClient?: ReturnType<typeof savedObjectsClientMock.create>;
 }): ManifestManager => {
   let cache = new LRU<string, Buffer>({ max: 10, maxAge: 1000 * 60 * 60 });
-  if (opts?.cache !== undefined) {
+  if (opts?.cache != null) {
```

Comment:
:grey_question: think you want `!==` instead of `!=`

Reply:
@bkimmel  Maybe I misunderstood, but I thought the prevailing recommendation was that `!= null` is preferable to `!== undefined` because it catches both `null` and `undefined`...

Reply:
It does, but coercively. `==` and it's `!=` cousin hurt to read. That fancy new nullish coalescer `??` could come in handy, since it's made to handle undefined/null like `if(opts?.cache ?? false)` maybe,